pkg/render/render_utils_test.go

```diff
@@ -91,3 +92,30 @@ func ExpectEnv(env []v1.EnvVar, key, value string) {
 	}
 	Expect(false).To(BeTrue(), fmt.Sprintf("Missing expected environment variable %s", key))
 }
+
+var (
+	tolerateMaster = corev1.Toleration{
```

Comment:
Is it necessary and beneficial to re-define these here?

Reply:
This test is a separate `_test` package, so the 'private' variables in the render package aren't accessible. I didn't want to  make the toleration variables external (by capitalizing) so I redefined.

I'm open to input on how to better set this up